src/Test/Utilities/CLRHelpers.cs

```diff
@@ -64,12 +65,14 @@ public static string[] PeVerify(string filePath)
             return PeVerify(File.ReadAllBytes(filePath), AppDomain.CurrentDomain.Id, filePath);
         }
 
-        private static readonly object Guard = new object();
+        private static readonly Mutex s_guard = new Mutex(false, new Guid().ToString());
```

Comment:
Hmm -- you're right. This is probably passing accidentally because locking a mutex takes so long.

However, locking with the same guid should work. I'll hardcode the GUID into the file.